single-property.php

```diff
@@ -17,9 +16,14 @@
     $address = get_post_meta($post->ID, '_igv_property_address', true);
     $summary = get_post_meta($post->ID, '_igv_property_summary', true);
     $gallery = get_post_meta($post->ID, '_igv_property_gallery', true);
-    $bottom_id = get_post_meta($post->ID, '_igv_property_bottom_image_id', true);
+
+    // Split $gallery into two arrays
+    $first_gallery = array_slice($gallery, 0, 2, true);
```

Comment:
`Warning: array_slice() expects parameter 1 to be array, string given` for empty gallery